File: qlty-cli/tests/lang/csharp/basic.in/NestedControl.cs
using System;

public class Main
{
    public static void NotNested(string foo, string bar)
    {
        if ((foo == "cat" && bar == "dog") || (foo == "dog" && bar == "cat"))
        {
            Console.WriteLine("Got a cat and a dog!");
        }
        else
        {
            Console.WriteLine("Got nothing");
        }
    }

    public static void F0(bool bar, bool baz, bool qux, bool quux)
    {
        if (bar)
        {
            if (baz)
            {
                if (qux)
                {
                    if (quux)
                    {
                        Console.WriteLine("Not deeply nested enough!");
                    }
                }
            }
        }
    }

    public static string F2(int foo)
    {
        switch (foo)
        {
            case 1:
                return "bar1";
            case 2:
                return "bar2";
            case 3:
                return "bar3";
            case 4:
                return "bar4";
            case 5:
                return "bar5";
            case 6:
                return "bar6";
            case 7:
                return "bar7";
            case 8:
                return "bar8";
            case 9:
                return "bar9";
            case 10:
                return "bar10";
            default:
                throw new ArgumentException("Invalid foo value");
        }
    }
}

File: qlty-cli/tests/lang/swift/basic.in/NestedControl.swift
func f1() {
    let bar = true
    let baz = true
    let qux = true
    let quux = true
    
    if bar {
        if baz {
            if qux {
                if quux {
                    print("Deeply nested!")
                }
            }
        }
    }
}

func f2(foo: Int) -> String {
    if foo == 1 {
        return "bar1"
    } else if foo == 2 {
        return "bar2"
    } else if foo == 3 {
        return "bar3"
    } else if foo == 4 {
        return "bar4"
    } else if foo == 5 {
        return "bar5"
    } else if foo == 6 {
        return "bar6"
    } else if foo == 7 {
        return "bar7"
    } else if foo == 8 {
        return "bar8"
    } else if foo == 9 {
        return "bar9"
    } else if foo == 10 {
        return "bar10"
    } else {
        fatalError("Invalid foo value")
    }
}